### PR TITLE
Do not use 'should' when describing tests

### PR DESCRIPTION
**Pull request by jcoyne**

### CODE REVIEW

File: spec/features/search_spec.rb [hunk 1]
```diff
@@ -111,7 +111,7 @@
     expect(page).to_not have_selector "#q[value='history']"
   end
 
-  it "should gracefully handle searches with invalid facet parameters" do
+  it "gracefullies handle searches with invalid facet parameters" do
```
Comment (cbeer) [inline spec/features/search_spec.rb]: `s/gracefullies/gracefully/`

Comment (jcoyne) [inline spec/features/search_spec.rb]: would you believe that this was done with a script?

Comment (cbeer) [inline spec/features/search_spec.rb]: 😱

**Comment by cbeer** [comment]
Rebase (and refix with the new changes from #1415?)

**Pull request merged**